config/sync/core.entity_form_display.media.embeddable.media_library.yml

```diff
@@ -0,0 +1,70 @@
+uuid: fd6f959e-edaf-4f32-a0c3-1655ccd3a342
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_form_mode.media.media_library
+    - field.field.media.embeddable.field_media_embeddable_code
+    - field.field.media.embeddable.field_media_embeddable_oembed
+    - media.type.embeddable
+  module:
+    - path
+id: media.embeddable.media_library
+targetEntityType: media
+bundle: embeddable
+mode: media_library
+content:
```

Comment:
Hide all the fields that aren't necessary.

Reply:
fixed!